final/introduction.tex

```diff
@@ -1,6 +1,33 @@
 \chapter{Introduction}
 \label{cha:introduction}
 
+Besides general purpose languages such as Java and C, nowadays there are also a
+lot of domain specific languages available. Domain specific languages can
+provide a lot of expressive power in a particular problem domain, thereby
+significantly easing the burden on the programmer. Well known examples of such
+DSLs include HTML for creating websites and SQL for dealing with relational
+data.
+
+At TU Delft significant effort has been spent to research and develop better
+domain specific languages. To facilitate prototyping and creation of DSLs the
+Spoofax Language Workbench was developed and first published in
+2010\cite{Kats10a}. Since then Spoofax has had several releases and will soon
+be arriving at version number 2.0.
+
+The description below is taken from \href{http://spoofax.org}{spoofax.org}:
+``The Spoofax Language Workbench supports the definition of all aspects of
```

Comment:
This appears out of nowhere and should probably be integration in the previous paragraph.